go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/docker/go-connections v0.3.0
 	github.com/docker/go-events v0.0.0-20170721190031-9461782956ad // indirect
 	github.com/docker/libnetwork v0.8.0-dev.2.0.20190604151032-3c26b4e7495e
-	github.com/godbus/dbus v4.1.0+incompatible // indirect
+	github.com/godbus/dbus v0.0.0-20151105175453-c7fdd8b5cd55 // indirect
```

Comment:
this is needed for the runc dep to compile. c7fdd8b5cd55 corresponds to the `v3` tag. I tried to give it a `v3.0.0-0.` prefix but doesn't seem to work even after I cleared the mod cache.

Reply:
```
$ go get github.com/opencontainers/runc@v1.0.0-rc8
...
/root/go/pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190321100706-95778dfbb74e/dbus/dbus.go:127:33: cannot use dbus.SystemBusPrivate (type func() (*dbus.Conn, error)) as type func(...<T>) (*dbus.Conn, error) in argument to dbusAuthHelloConnection
/root/go/pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190321100706-95778dfbb74e/dbus/dbus.go:136:33: cannot use dbus.SessionBusPrivate (type func() (*dbus.Conn, error)) as type func(...<T>) (*dbus.Conn, error) in argument to dbusAuthHelloConnection
/root/go/pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190321100706-95778dfbb74e/dbus/dbus.go:146:42: undefined: dbus.ConnOption
/root/go/pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190321100706-95778dfbb74e/dbus/dbus.go:204:48: undefined: dbus.ConnOption
/root/go/pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190321100706-95778dfbb74e/dbus/dbus.go:224:53: undefined: dbus.ConnOption
# github.com/opencontainers/runc/libcontainer/specconv
/root/go/pkg/mod/github.com/opencontainers/runc@v1.0.0-rc8/libcontainer/specconv/spec_linux.go:250:26: spec.Linux.IntelRdt.MemBwSchema undefined (type *specs.LinuxIntelRdt has no field or method MemBwSchema)
/root/go/pkg/mod/github.com/opencontainers/runc@v1.0.0-rc8/libcontainer/specconv/spec_linux.go:251:54: spec.Linux.IntelRdt.MemBwSchema undefined (type *specs.LinuxIntelRdt has no field or method MemBwSchema)
```

Reply:
I should have just used `go get -d`